themes/min-dark.json

```diff
@@ -2,6 +2,8 @@
 	"name": "Min Dark",
 	"type": "dark",
 	"colors": {
+		"editorIndentGuide.activeBackground": "#7D7D7D",
+        	"editorIndentGuide.background": "#383838",
```

Comment:
Might be more subtle to set this to 2A2A2A ?

Reply:
After more testing I quite enjoy using  
```
        "editorIndentGuide.activeBackground": "#383838",
        "editorIndentGuide.background": "#2A2A2A",
```
instead.

It's more subtle and less in your face